bin/dev-setup

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env node
+
+var mongoose = require('mongoose');
+var mongoConfig = require('../lib/mongoConfig');
+var Song = require('../models/song');
+
+var sampleSong = new Song({
+  title: 'Amazing Grace',
+  author: 'John Newton and Virginia Harmony',
+  year: '1779',
+  lang: 'English',
+  translator: null,
+  contributor: 'dev-setup-script',
+  copyright: 'Public',
+  lyric: [
+    ['Amazing grace! how sweet the sound,',
+    'That saved a wretch like me!',
+    'I once was lost, but now am found;',
+    'Was blind, but now I see.'],
+    ['The earth shall soon dissolve like snow;',
+    'The sun forbear to shine;',
+    'But God, who called me here below,',
+    'Will be forever mine.']
+  ],
+  source: null,
+  oriSong: null,
```

Comment:
i think we dont need this field anymore. i already put a github issue regarding this